Tests/TuistKitTests/ProjectEditor/ProjectEditorMapperTests.swift

```diff
@@ -117,13 +147,17 @@ final class ProjectEditorMapperTests: TuistUnitTestCase {
         let (project, graph) = try subject.map(tuistPath: tuistPath,
                                                sourceRootPath: sourceRootPath,
                                                xcodeProjPath: xcodeProjPath,
+                                               setupPath: nil,
+                                               configPath: nil,
                                                manifests: manifestPaths,
                                                helpers: helperPaths,
                                                templates: templates,
                                                projectDescriptionPath: projectDescriptionPath)
 
         // Then
         let targetNodes = graph.targets.values.flatMap { targets in targets.compactMap { $0 } }.sorted(by: { $0.target.name < $1.target.name })
+
+        // expecting `targetNodes == [TemporaryDirectory.XXXMainifest]`
```

Comment:
nitpick: alternatively, instead of adding such a comment, you can add `message` to `XCTAssertEqual()` such as `XCTAssertEqual(..., "Expected [TemporaryDirectory.XXXManifest]")`